From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Date: Tue, 17 Dec 2013 11:08:10 +0800
Subject: sfc: fix sparse non static symbol warning
Patch-mainline: v3.14-rc1
Git-commit: d2adcaa82d0c9d6423afcbfcb68658ff86794c29
References: bnc#856380 FATE#315942

Fixes the following sparse warning:

drivers/net/ethernet/sfc/falcon.c:2601:6: warning:
 symbol 'falcon_pull_nic_stats' was not declared. Should it be static?

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/sfc/falcon.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
--- a/drivers/net/ethernet/sfc/falcon.c
+++ b/drivers/net/ethernet/sfc/falcon.c
@@ -2614,7 +2614,7 @@ void falcon_start_nic_stats(struct efx_n
 /* We don't acutally pull stats on falcon. Wait 10ms so that
  * they arrive when we call this just after start_stats
  */
-void falcon_pull_nic_stats(struct efx_nic *efx)
+static void falcon_pull_nic_stats(struct efx_nic *efx)
 {
 	msleep(10);
 }
